Makefile.am

```diff
@@ -119,18 +116,19 @@ if USE_TESTS
 noinst_PROGRAMS += tests
 tests_SOURCES = src/tests.c
 tests_CPPFLAGS = $(SECP_INCLUDES) $(SECP_TEST_INCLUDES) $(SECP_CONFIG_DEFINES)
-if VALGRIND_ENABLED
```

Comment:
fc2ff99cfdc7005598826d0485aabe3abb349525 removes the last usage of the `VALGRIND_ENABLED`. The following line can be removed now as well https://github.com/bitcoin-core/secp256k1/blob/31ed5386e8450845c2cdbf1af9dd257937c46e48/configure.ac#L228